automation/unit-tests.sh

```diff
@@ -43,6 +43,12 @@ LATEST_CT=$(curl -L https://api.github.com/repos/kubevirt/common-templates/relea
             jq '.[] | select(.prerelease==false) | .name' | sort -V | tail -n1 | tr -d '"')
 oc apply -f https://github.com/kubevirt/common-templates/releases/download/${LATEST_CT}/common-templates-${LATEST_CT}.yaml
 
+python3 automation/validate-pvc-name-stability.py
+if [ $? -eq 1 ];then
+  echo "[Upgrade] Validation of PVC name stability failed"
+  exit 1
```

Comment:
Related to previous note: If we will exit on any non-zero, we should pass that value through here so it may be consumed by the caller.

Reply:
Do the two comments also apply to the other uses inside this file?
Should I change them as well?